java/core/src/main/java/sleeper/core/statestore/transactionlog/transaction/impl/InitialisePartitionsTransaction.java

```diff
@@ -29,8 +30,8 @@
 import java.util.Objects;
 
 /**
- * A transaction to set all partitions in a Sleeper table. This should specify the whole partition tree. Any partitions
- * that were present before will be deleted.
+ * Sets all partitions in a Sleeper table. These should build into a complete partition tree, where all partitions are
+ * linked to a single root. Any partitions that were present before will be deleted.
  */
```

Comment:
This would be clearer as "where there is a single root and all partitions are connected to that either directly or indirectly via other partitions".

Reply:
Done, thanks.